### PR TITLE
[Docs] Extend the tutorial

### PR DESCRIPTION
**Pull request by filipsobol**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: Extend the step-by-step tutorial for new users. See #14583.

---

:warning: This PR is continuation of #14637. Before reviewing and merging this PR, merge #14637 and merge the latest `master` to this branch :warning:

### CODE REVIEW

File: docs/tutorial/events-and-observables.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+```js
+const button = new ButtonView( locale );
+```
+
+If you want to make a custom class an observable, see the {@link framework/deep-dive/observables Observables} document.
+
+### Setting the observable properties
+
+Then, we called the `.set()` method to update few button properties. Calling this method fires an event that can also be listened to, if necessary.
```
Comment (niegowski) [inline docs/tutorial/events-and-observables.md:41]: This is more like a batch set in this case. In the general case, this marks properties as observable. 

I'd suggest not to confuse users with the `set` event. We use it almost never.

File: docs/tutorial/events-and-observables.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+button.bind( 'isOn', 'isEnabled' ).to( command, 'value', 'isEnabled' );
+```
+
+The above line will attach listeners that listen for changes of the `value` and `isEnabled` properties of the command, and update the `isOn` and`isEnabled` button properties accordingly.
+
+This is what the code would look like if you used the event system directly, without using the `bind(...).to(...)` abstraction provided by the observables:
+
+```js
+command.on( 'change:value', ( event, propName, newValue, oldValue ) => {
+	button.set( 'isOn', newValue );
```
Comment (niegowski) [inline docs/tutorial/events-and-observables.md:81]: The observable property has a setter (after initial call to `.set( ... )`).
```suggestion
	button.isOn = newValue;
```

File: docs/tutorial/events-and-observables.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+} );
+
+command.on( 'change:isEnabled', ( event, propName, newValue, oldValue ) => {
+	button.set( 'isEnabled', newValue );
```
Comment (niegowski) [inline docs/tutorial/events-and-observables.md:85]:
```suggestion
	button.isEnabled = newValue;
```

File: docs/tutorial/keystrokes.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+Currently, our `highlight` plugin requires the user to click the button in the editor's toolbar to highlight selected text. This can be a problem for users who rely on the keyboard, so let's add a keyboard shortcut as an alternative way to highlight the text.
+
+## Adding keyboard shortcuts
+
+Common shortcut for highlighting text is <kbd>CTRL</kbd> + <kbd>ALT</kbd> + <kbd>H</kbd>, so that's what we're going to use in our plugin.
+
+To execute the `highlight` command when those keys are pressed, add the following code to the end of the `Highlight` function:
+
+```js
+editor.keystrokes.set( 'Ctrl+Alt+H', 'highlight' );
```
Comment (niegowski) [inline docs/tutorial/keystrokes.md:20]: Maybe we should mention that keystrokes on MacOS are automatically translated to `Cmd+Alt+H'?

**Comment by godai78** [comment]
The work on this PR has been moved to this one: https://github.com/ckeditor/ckeditor5/pull/14807

**Pull request closed without merging**